packages/trader/src/AppV2/Components/Guide/decription-modal.tsx

```diff
@@ -0,0 +1,115 @@
+            CONTRACT_LIST['MATCHES/DIFFERS'],
+            CONTRACT_LIST['EVEN/ODD'],
+            CONTRACT_LIST['OVER/UNDER'],
+        ];
+
+        const toggleVideoPlayer = (e?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => {
+            clickAndKeyEventHandler(() => setIsVideoPlayerOpened(!is_video_player_opened), e);
+        };
+
+        const ordered_contract_list = [...AVAILABLE_CONTRACTS].sort(
+            (a, b) => order.findIndex(item => item === a.id) - order.findIndex(item => item === b.id)
+        );
+
+        React.useEffect(() => {
+            if (modal_ref.current) is_video_player_opened ? modal_ref.current.showModal() : modal_ref.current.close();
+        }, [is_video_player_opened]);
+
+        return (
+            <React.Fragment>
+                <ActionSheet.Root isOpen={is_open} onClose={onClose} position='left'>
+                    <ActionSheet.Portal shouldCloseOnDrag>
+                        <ActionSheet.Content className='guide__wrapper'>
+                            <Heading.H4 className='guide__title'>
+                                <Localize i18n_default_text='Trade types' />
+                            </Heading.H4>
+                            <div className='guide__menu'>
+                                {ordered_contract_list.map(({ tradeType, id }) => (
+                                    <Chip.Selectable
+                                        key={id}
+                                        onChipSelect={onChipSelect}
+                                        selected={id === selected_contract_type}
+                                    >
+                                        <Text size='sm'>{tradeType}</Text>
+                                    </Chip.Selectable>
+                                ))}
+                            </div>
+                            <div className='guide__contract-description' key={selected_contract_type}>
+                                <TradeDescription contract_type={selected_contract_type} onTermClick={onTermClick} />
+                                <VideoPreview
+                                    contract_type={selected_contract_type}
+                                    toggleVideoPlayer={toggleVideoPlayer}
+                                    video_src={video_src}
+                                />
+                            </div>
+                        </ActionSheet.Content>
+                        <ActionSheet.Footer
+                            alignment='vertical'
+                            primaryAction={{
+                                content: <Localize i18n_default_text='Got it' />,
+                                onAction: onClose,
+                            }}
+                            className='guide__button'
+                        />
+                    </ActionSheet.Portal>
+                </ActionSheet.Root>
+                {is_video_player_opened && (
+                    <dialog
```

Comment:
I decided not to create another modal-like component, by try to use native `dialog` tag for this purpose.
<img width="1435" alt="Screenshot 2024-07-04 at 11 57 12 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/033a5c54-574a-4048-8b2b-ace05d73a12e">